### DISCUSSION TITLE
Error in updating certificate.

### DISCUSSION THREAD

**Discussion by piqm** [rcl-ssl/docs#14] [Q&A] [answered]
I have this error, how can I solve it?  Error in updating certificate. AADSTS9002332: Application '797f4846-ba00-4fd7-ba43-dac1f8f63013'(Windows Azure Service Management API) is configured for use by Azure Active Directory users only. Please do not use the /consumers endpoint to serve this request.

**Comment by rcladmin** [comment, marked as answer]
This error occurs when you sign in to the RCL portal with a Microsoft Personal Account (MSA). You will need to create an AAD User Account, Set Access Control to your Azure Subscription and associate the AAD User account with your RCL subscription. You then need to sign in to the RCL portal with the AAD User account allow consent  and perform the update operation. Follow the instructions in the link below 

https://docs.rclapp.com/authorization/sign-in-accounts.html
